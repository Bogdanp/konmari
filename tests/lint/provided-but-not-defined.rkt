#lang racket/base

(provide
 g
 (rename-out [h i]
             [j k])
 x)

(define x 42)
