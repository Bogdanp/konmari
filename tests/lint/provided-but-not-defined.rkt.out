/provided-but-not-defined.rkt:7:17:error:not an identifier
/provided-but-not-defined.rkt:7:15:error:identifier 'l' provided but not defined
/provided-but-not-defined.rkt:6:15:error:identifier 'j' provided but not defined
/provided-but-not-defined.rkt:5:15:error:identifier 'h' provided but not defined
/provided-but-not-defined.rkt:4:2:error:identifier 'g' provided but not defined
